docs/js-guide.md

```diff
@@ -103,6 +103,36 @@ function getPDFTitle() {
 class PDFIntegration { ... };
 ```
 
+#### Ref naming
+
+Preact does not allow the `ref` prop to be passed verbatim on non-HTML components. Instead it
```

Comment:
```suggestion
Preact does not allow the `ref` prop to be accessed by non-HTML components. Instead it
```

_Verbatim_ refers specifically to the wording of some text not being changed, eg. when quoted or copied. In this context the `ref` prop can be set on a non-HTML component, but the component can't currently access it.

Reply:
Makes sense